src/ui_basic/unique_window.h

```diff
@@ -63,7 +63,9 @@ struct UniqueWindow : public Window {
 	             Registry*,
 	             int32_t w,
 	             int32_t h,
-	             const std::string& title);
+	             const std::string& title,
+	             int32_t x = 0,
+	             int32_t y = 0);
```

Comment:
This is inconsistent with other UI element constructors, where the x/y coords are always set just before the width and height. How about moving them up to between the `Registry*` and the `w`/`h` parameters, and adding a second ctor without these two parameters that just invokes the larger ctor?

```c++
	UniqueWindow(Panel* parent,
	             WindowStyle,
	             const std::string& name,
	             Registry*,
	             int32_t x,
	             int32_t y,
	             int32_t w,
	             int32_t h,
	             const std::string& title);

	UniqueWindow(Panel* parent,
	             WindowStyle s,
	             const std::string& name,
	             Registry* r,
	             int32_t w,
	             int32_t h,
	             const std::string& title)
	: UniqueWindow(parent, s, name, r, 0, 0, w, h, title) {}
```